app-agent/src/options.rs

```diff
@@ -56,7 +56,7 @@ pub mod cli {
     #[derive(Args, Clone)]
     pub struct CommonArgs {
         /// Path to the config file.
-        #[arg(long, default_value = "alumet-config.toml")]
+        #[arg(long, env, default_value = "alumet-config.toml")]
```

Comment:
I think it's risky to use CONFIG to store the config path for alumet

Reply:
Alumet won't be the only software running on the device i.e. probably not the only one to use CONFIG

Reply:
You should use ALUMET_CONFIG or something like that

Reply:
yes, good point

Reply:
The following will probably work: `env("ALUMET_CONFIG")`